.github/workflows/case-data-export-prod.yml

```diff
@@ -0,0 +1,38 @@
+name: Export prod case data
+
+on:
+  schedule:
+    # Every day at 6AM UTC (data is typically ingested at 5AM UTC).
```

Comment:
This is done as an action in a different repo, right? If it's the same repo, we can trigger this off of the newly added `workflow_run`.

https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_run

Reply:
oh cool! we could use that but given that we're probably going to stop running that workflow starting next week (whenever we drop the data from mongodb & stop re-importing), it might not make sense. ultimately this is really "arbitrarily running once a day at 6am because why not"

Reply:
SGTM